docs/gadget-devel/gadget-wasm-api-raw.md

```diff
@@ -420,3 +434,13 @@ Parameters:
 
 Return value:
 - 0 in case of success, 1 otherwise.
+
+#### `mapClose(m uint32) uint32`
+
+Close the map created by `newMap()`.
```

Comment:
```suggestion
Close the map created by `newMap()`. The map handle is released and can no longer be used.
```

I wonder, in that case, if it should be called `mapRelease`, like other functions `dataSourceRelease`, `dataArrayRelease`...

Reply:
I think it should be called `mapClose`. `dataSourceRelease` and `dataArrayRelease` are related to the [datasource.Release](https://github.com/inspektor-gadget/inspektor-gadget/blob/13440171913d6d48f22e2cc034eb7a330e13a990/pkg/datasource/datasource.go#L142-L143)